laghos.cpp

```diff
@@ -663,15 +691,24 @@ int main(int argc, char *argv[])
             MPI_Reduce(&mem, &mmax, 1, MPI_LONG, MPI_MAX, 0, pmesh->GetComm());
             MPI_Reduce(&mem, &msum, 1, MPI_LONG, MPI_SUM, 0, pmesh->GetComm());
          }
+         // const double internal_energy = hydro.InternalEnergy(e_gf);
+         // const double kinetic_energy = hydro.KineticEnergy(v_gf);
          if (mpi.Root())
          {
             const double sqrt_norm = sqrt(norm);
+
             cout << std::fixed;
             cout << "step " << std::setw(5) << ti
                  << ",\tt = " << std::setw(5) << std::setprecision(4) << t
                  << ",\tdt = " << std::setw(5) << std::setprecision(6) << dt
                  << ",\t|e| = " << std::setprecision(10) << std::scientific
                  << sqrt_norm;
+            //  << ",\t|IE| = " << std::setprecision(10) << std::scientific
```

Comment:
Again, use a condition rather than commenting out.

Reply:
These changes made to the main directory are actually merged from the `master` branch. I guess we should better not make any changes so that when we merge from the `master` in the future, it would not cause any conflicts.